linking/static/aplib/linked-against-aplib.yml

```diff
@@ -0,0 +1,14 @@
+rule:
+  meta:
+    name: linked against aPLib
+    namespace: linking/static/aplib
+    authors:
+      - still@teamt5.org
+    scope: file
+    mbc:
+      - Data::Compression Library [C0060]
+    examples:
+      - 72c8e3c8049927fa1dd53b61d9b67b9d74ab5b6c030bc38dfbe9a338a32438c9
```

Comment:
can you upload this to the capa-testfiles repo, please?

Reply:
Done